### PR TITLE
New Feature: Small fixes to the templates and wiki image

### PR DESCRIPTION
**Pull request by Thomas-Boi**
# **New feature section**

## Double check these details before you open a PR

- [ x ] PR does not match another non-stale PR currently opened
- [ x ] PR name matches the format *New Feature: a brief description of feature*

## This PR adds/fixes

Add a checkbox to the issue forms to ensure people check previous issues before opening a new one. Also cropped one of the wiki image so content is clearer

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]:
```suggestion
        - label: Confirm
    validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/feature_request.yml:14]:
```suggestion
        - label: Confirm
    validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/icon_request.yml [hunk 3]
```diff
@@ -1,26 +1,36 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/icon_request.yml:14]:
```suggestion
        - label: Confirm
    validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 4]
```diff
@@ -1,22 +1,28 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/other.yml:14]:
```suggestion
        - label: Confirm
    validations:
      required: true
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great work on this! 🚀

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Thomas-Boi) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: Hi Pan, I'm not too sure what's the change here. Everything looks the same to me. Was it an end line or something?

Comment (Panquesito7) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: No 😅 This will make the checkbox obligatory to mark (see `validations`, `required: true`).

Comment (Thomas-Boi) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: Ah I see, I was confused by the color coding. I tried to do that before too but GitHub doesn't support this [attribute](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes). I can however make that specific box mandatory.

Comment (Panquesito7) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: > Ah I see, I was confused by the color coding. I tried to do that before too but GitHub doesn't support this [attribute](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes). I can however make that specific box mandatory.

Yes, that'd be great, thanks! 👍

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by Thomas-Boi** [comment]
I think I'll just merge the PR. It's a simple one anyway

**Pull request merged**